### PR TITLE
fix(card-section-carousel): removed page size knob

### PR DESCRIPTION
**Pull request by NSDrowned**
### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/5329

### Description

Removed "page size knob" and any mention of number of cards from docs

### Changelog

**Removed**

- Removed page size knob
- Removed `page-size=3` from docs

### CODE REVIEW

**Review by shixiedesign — COMMENTED** [review]
Still seeing in React Wrapper, under: 
- Card section carousel
- Carousel -- default
- Carousel -- Center 8 columns

Apr-27-2021 09-16-13

File: packages/web-components/src/components/card-section-carousel/__stories__/card-section-carousel.stories.ts [hunk 1]
```diff
@@ -41,8 +39,8 @@ const Card = ({ copy = copyDefault, heading = headingDefault, href = hrefDefault
   </dds-card>
 `;
 
-export const Default = ({ parameters }) => {
-  const { pageSize } = parameters?.props?.Carousel ?? {};
+export const Default = () => {
+  const pageSize = null;
```
Comment (kennylam) [inline packages/web-components/src/components/card-section-carousel/__stories__/card-section-carousel.stories.ts:43]: @NSDrowned Is this still needed?

Comment (NSDrowned) [reply packages/web-components/src/components/card-section-carousel/__stories__/card-section-carousel.stories.ts:43]: You're right, not sure why I left that there... 🤦‍♂️

**Comment by NSDrowned** [comment]
@shixiedesign, fixed react wrapper 👍

**Review by kennylam — APPROVED** [review]
Looks good thanks @NSDrowned!

**Review by shixiedesign — APPROVED** [review]
Thanks!

**Pull request merged**